django/db/backends/postgresql/operations.py

```diff
@@ -365,6 +367,9 @@ def subtract_temporals(self, internal_type, lhs, rhs):
 
     def explain_query_prefix(self, format=None, **options):
         extra = {}
+        if serialize := options.pop("serialize", None):
+            if serialize.upper() in ["TEXT", "BINARY"]:
```

Comment:
```suggestion
            if serialize.upper() in {"TEXT", "BINARY"}:
```

"Membership testing with sets and dictionaries is much faster, O(1), than searching sequences, O(n). When testing "a in b", b should be a set or dictionary instead of a list or tuple."
See 
https://wiki.python.org/moin/PythonSpeed#Use_the_best_algorithms_and_fastest_tools

In this case the list is very small and you can't see differences in performance, but I suggest to  use the fastest tools when you have it and they are free.

Reply:
Thank you for the suggestion @pauloxnet - I do agree with the sentiment of using the fastest tools when they're available!